src/reducers/index.js

```diff
@@ -2,7 +2,7 @@
 import { combineReducers } from 'redux'
 import lang from './lang'
 import authenticate from './authenticate'
-import {removeAllContent, setMyItems, removeMyItem, setMySelections} from './inputs'
+import {removeAllContent, setMyItems, removeMyItem, setMySelections } from './inputs'
```

Comment:
If adding a space before the closing brace, considering adding a space after the opening brace?

Reply:
Fixed in the latest commit.